apps/web/app/page.tsx

```diff
@@ -48,6 +39,15 @@ const Page = async () => {
     console.error("Failed to get first environment of user; signing out");
     return <ClientLogout />;
   }
+  const userOrganizations = await getOrganizationsByUserId(session.user.id);
+
+  if (userOrganizations.length === 0) {
+    return redirect("/setup/organization/create");
+  }
+
+  if (!ONBOARDING_DISABLED && !session.user.onboardingCompleted) {
```

Comment:
I think we should have the check in the environment itself and not on the root level.
Since this is the product onboarding; if you try to access and environment where the product onboarding wasn't completed, you should go through the Onboarding.
This should also be independent of the `ONBOARDING_DISABLED` env variable. Maybe since we now don't have a user onboarding anymore, we don't need it?
(The reason for the variable was to skip the onboarding for users that signed up and auto-added to an organization)